Source/Core/Core/IOS/SDIO/SDIOSlot0.h

```diff
@@ -132,10 +142,19 @@ class SDIOSlot0 : public Device
                      u32 BufferInSize2, u32 _BufferOut, u32 BufferOutSize);
   void OpenInternal();
 
+  u32 GetOCRegister() const;
+
+  void GetCSDv1(u32 csd[4]);
```

Comment:
Can these not just be modified to return a `std::array<u32, 4>` instead of using an out parameter?